server/app/views/questiontypes/PhoneQuestionRenderer.java

```diff
@@ -0,0 +1,86 @@
+package views.questiontypes;
+
+import static j2html.TagCreator.*;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import j2html.tags.specialized.DivTag;
+import play.i18n.Messages;
+import services.MessageKey;
+import services.Path;
+import services.applicant.ValidationErrorMessage;
+import services.applicant.question.ApplicantQuestion;
+import services.applicant.question.PhoneQuestion;
+import views.components.FieldWithLabel;
+import views.components.SelectWithLabel;
+import views.style.ReferenceClasses;
+
+public class PhoneQuestionRenderer extends ApplicantSingleQuestionRenderer {
+  PhoneQuestionRenderer(ApplicantQuestion question) {
+    super(question);
+  }
+
+  @Override
+  public String getReferenceClass() {
+    return "cf-question-phone";
+  }
+
+  @Override
+  protected DivTag renderInputTag(
+      ApplicantQuestionRendererParams params,
+      ImmutableMap<Path, ImmutableSet<ValidationErrorMessage>> validationErrors,
+      ImmutableList<String> ariaDescribedByIds) {
+    PhoneQuestion phoneQuestion = question.createPhoneQuestion();
+
+    Messages messages = params.messages();
+
+    SelectWithLabel countryCodeField =
+        (SelectWithLabel)
```

Comment:
**nitpick:** 
```suggestion
```

Reply:
Can't remove this unfortunately. I see this error: 
Required type: SelectWithLabel
Provided: FieldWithLabel